src/main/java/org/javarosa/core/model/instance/FormInstance.java

```diff
@@ -175,6 +175,7 @@ public FormInstance clone() {
     public void readExternal(DataInputStream in, PrototypeFactory pf) throws IOException, DeserializationException {
         super.readExternal(in, pf);
         schema = (String)ExtUtil.read(in, new ExtWrapNullable(String.class), pf);
+        formVersion = (String)ExtUtil.read(in, new ExtWrapNullable(String.class), pf);
```

Comment:
can we also do the same for `uiVersion` as that also seems to be missing from here.